_overview/fiat-permissions-overview.md

```diff
@@ -0,0 +1,240 @@
+
+## Continuous integration system
+
+`build1` is a Jenkins deployment used for CI in this example. The Halconfig for controlling access to `build1` looks similar to the following snippet:
+
+```
+ci:
+  jenkins:
+    enabled: true
+    masters:
+    - name: build1
+      permissions:
+        READ:
+        - admin
+        - dev
+        - qa
+        - ops
+        WRITE:
+        - admin
+        - ops
+```
+
+## Applications
+
+`app1` is one of the applications that needs to be deployed. Configuring permissions for an application is done in Deck, Spinnaker's UI, when you create or edit an application:
+
+![app1 permissions](/images/fiat_overview_app1_perms.png)
+
+`app2`, `app3`, and `app4` will look slightly different since they have different permissions based on the mapping exercise. 
+
+For information about how to configure permissions for applications, see [Applications](https://www.spinnaker.io/setup/security/authorization/#applications).
+
+## Applying changes
+
+Whenever you make a change to permissions, run `hal deploy apply` to apply your changes to the Spinnaker deployment.
+
+## Verifying permissions
+
+You can verify what permissions are assigned to a role at any time.
+
+### Admin permissions
+
+Check `fiat-local.yml` to see who is assigned an `admin` role.
+
+### Permissions in Halconfig
+
+In your Halconfig, you can verify several sets of permissions.
+
+For example, search for `ci` to find the continuous integration section. Look for the `permissions` key and examine the `READ` and `WRITE` subkeys. All the user roles that have read or write permission for the CI system are listed here. The same thing can be done for Clouddriver accounts.
+
+### Permissions for apps
+
+Check the permissions for all applications in Spinnaker with a REST API call to Gate.
+
+**Headers** 
+
+|  Request URL  |  `$GATE_URL/applications`  |
+|  Request Method  |  `GET`  |
+|  content-type  |  `application/json;charset=UTF-8`  |
+
+The API call returns information about the apps. Refer to the `name` and `permissions` sections to find your applications and the corresponding permissions:
+
+```
+{
+    ...
+    "name": "app2",
+    "permissions": {
+          "EXECUTE": [
+            "admin",
+            "dev",
+            "ops"
+          ],
+          "READ": [
+            "admin",
+            "dev",
+            "ops"
+          ],
+          "WRITE": [
+            "admin",
+            "dev"
+          ]
+    },
+    ...
+}
+```
+
+### Permissions for Spinnaker service accounts
+
+Verifying the permissions for service accounts requires access to the Front50 and Fiat pods.
+
+List all the service accounts with the following command (from the Front50 pod):
+
+```
+export FRONT50=http://spin-front50:8080
+curl -s $FRONT50/serviceAccounts
+```
+
+Check user or service account permissions for all of Spinnaker (from the Fiat pod):
+
+```
+export FIAT=http://spin-fiat:7003
+curl -s $FIAT/authorize/$user-or-service-account
+```
```

Comment:
So I discovered that you can audit any user in Spinnaker regardless of whether they are a service account or not by running `curl -s $FIAT/authorize/$user-or-service-account`. Not sure if we should make not of this here or not.

Reply:
Doesn't the intro sentence cover that by saying:
`Check user or service account permissions for all of Spinnaker (from the Fiat pod):`